website/templates/emails/pending_embargo_admin.html.mako

```diff
@@ -3,36 +3,51 @@
 <%def name="content()">
 <tr>
   <td style="border-collapse: collapse;">
+    <%!from website import settings%>
     Hello ${user.fullname},
     <p>
     % if is_initiator:
-        You initiated an embargoed registration of ${project_name} with embargo end date ${embargo_end_date.date()}.
-		The proposed registration can be viewed here: ${registration_link}.
+      You have requested final approvals to submit your registration
+      titled <a href="${registration_link}">${reviewable.title}</a>.
     % else:
-        ${initiated_by} initiated an embargoed registration of ${project_name} with embargo end date ${embargo_end_date.date()}.
-		The proposed registration can be viewed here: ${registration_link}.
+      ${initiated_by} has requested final approvals to submit your registration
+      titled <a href="${registration_link}">${reviewable.title}</a>.
     % endif
+    </p>
     <p>
     % if is_moderated:
-        If approved, an embargoed registration will be created for the project and sent to ${reviewable.provider.name}
-         moderators for review.
+      If approved by all admin contributors, the registration will be submitted for moderator review.
+      If the moderators approve, the registration will be embargoed until ${embargo_end_date.date()},
+      at which point it will be made public as part of the {reviewable.provider.name} registry.
     % else:
-        If approved, a registration will be created for the project, and it will remain private until it is withdrawn,
-         it is manually made public, or the embargo end date is passed on ${embargo_end_date.date()}.
+      If approved by all admin contributors, the registration will be embargoed until ${embargo_end_date.date()},
+      at which point it will be made public as part of the {reviewable.provider.name} registry.
     % endif
+    </p>
+    <p style="color:red;">
+    You have ${approval_time_span} hours from midnight tonight (EDT) to approve or cancel
```

Comment:
do we really use EDT, daylight savings and everything? i wanted to say we should give an explicit timezone offset like "UTC-04:00", but that'd be annoying if it's not constant